src/mopi/run_inference.py

```diff
@@ -31,15 +32,15 @@ def inference(
         plugins=[OutputAnalyserPlugin()],
     )
 
-    store, pipeline = runner.infer()
+    store, pipeline, _ = runner.infer()
 
     return experiment, pipeline, store
 
 
 def run_inference(pipeline: Pipeline, texts: List[str]) -> Tuple[int, float]:
 
     text_with_fake_labels = [[text, 0] for text in texts] + [["dummy_text", 1]]
-
+    project_name = pipeline.run_context.project_name
```

Comment:
We need to get the original project name, because that is how we store the models in a directory.